spec/dealership_spec.rb

```diff
@@ -0,0 +1,60 @@
+require './lib/car'
+require './lib/dealership'
+require 'pry'
+require 'rspec'
+
+RSpec.describe Dealership do
+  it 'exist' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership).to be_instance_of(Dealership)
+  end
+
+  it 'has an inventory' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory).to eq([])
+  end
+
+  it "how many items are in inventory" do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.inventory_count).to eq(0)
+  end
+
+  it "adds cars to inventory" do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    #binding.pry
+    expect(dealership.inventory)
+  end
+
+  it 'counts inventory after adding cars' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    car_1 = Car.new("Ford Mustang", 1500, 36)
+    car_2 = Car.new("Toyota Prius", 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    expect(dealership.inventory_count).to eq(2)
+  end
+
+  it 'have inventory?' do
+    dealership = Dealership.new("Acme Auto", "123 Main Street")
+    expect(dealership.has_inventory?).to be(false)
```

Comment:
This test passes fine, and your code works, this is just a piece of advice: 
When your test is called "has inventory", but you only test for `false`, how can you prove that it **has** inventory? 

You'd need an assertion somewhere (an `expect`) that tests for `true`. That just means adding a car to the Dealership's inventory, and writing an expect has_inventory to be true. Example: 

```
dealership.add_car(car_1)
expect(dealership.has_inventory?).to be true
```

Reply:
This would ensure you have fully tested the method. But as I said, your logic in the code works out well here, just keep this in mind for the future. :)